libtransact/src/protocol/batch.rs

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2018 Bitwise IO, Inc.
+ * Copyright 2021 Cargill Incorporated
```

Comment:
copyright updates should be added to the other commit that updates all of the files touched in this PR

Reply:
They are included in the handler fix commit

Reply:
This was intentional because the copyright addition is new instead of being an update.